azure-migrate/dependencies-at-scale/AzMig_Dependencies.ps1

```diff
@@ -0,0 +1,585 @@
+
+function GetRequestProperties()
+{
+
+    $ErrorActionPreference = 'Stop'
+    
+
+    if(-not (Get-Module Az.Accounts)) {
+        Import-Module Az.Accounts
+    }
+    
+    if ((Get-Module Az.Accounts).Version -lt "2.2.0") {
+        throw "At least Az.Accounts 2.2.0 is required, please update before continuing."
+    }
+    
+    $CurrentContext = Get-AzContext
+    if (-not $CurrentContext) {
+        throw "Not logged in. Use Connect-AzAccount to log in"
+    }    
+  
+    $TenantId = $CurrentContext.Tenant.Id
+    $UserId = $CurrentContext.Account.Id
+    if ((-not $TenantId) -or (-not $UserId)) {
+        throw "Tenant not selected. Use Select-AzSubscription to select a subscription"
```

Comment:
```suggestion
        throw "Tenant not selected. Use Set-AzContext to select a subscription"
```

Reply:
`Select-AzSubscription` is an alias to the `Set-AzContext`. Specify the cmdlet name instead of the alias.